src/server/object_services/object_server.js

```
@@ -2163,24 +2208,27 @@ function _sort_parts_by_seq(a, b) {
+        const obj_md = objects[i].data;
+        const reply = _get_delete_obj_reply(obj_md);
+        const indices = object_version_index_map[`${obj_md.key}::${version_id}`];
 
-        if (!obj_md) {
-            results[idx] = obj ?? {};
-            continue;
+        for (const j of indices) {
+            reply.seq = await MDStore.instance().alloc_object_version_seq();
+            results[j] = reply;
         }
+    }
```

ESLint

[error] The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.

(guard-for-in)